### ISSUE TITLE
Mod breaks Jar in Jar loading on Forge 1.19.2

### ISSUE THREAD

**Issue by Gronglegrowth** [wendall911/ChargedCharms#2]
When version 1.7.0.1 of this mod is loaded, other mods which rely on jar in jar will fail to load, because somehow this mod is breaking jij loading. This can be evidenced by loading the game with both this and the Thermal series, which utilizes jij.

**Comment by wendall911** [comment]
I'll take a look and see what the issue might be.

**Comment by wendall911** [comment]
Ok, you'll have to give a specific mod that is causing a conflict. I tried a couple of the thermal series, foundation and dynamics and I don't see the issue. We have a half dozen other mods using jarJar from Forge and it works correctly. I actually loaded several other jarJar mods and see no conflicts. Create, Oh THe Biomes You'll Go and Charm of Undying (uses same jij) and no issues.

Maybe it is tied to the Forge version you're using? At any rate, I need more information. I'm happy to debug and provide a fix, but I'm using standard jarJar provided by Forge.
